Resources/Prototypes/DeltaV/Entities/Structures/Doors/Windoors/windoor.yml

```diff
@@ -21,6 +21,32 @@
   id: WindoorSecureParamedicLocked
   suffix: Paramedic, Locked
   components:
+  - type: AccessReader
+    access: [["Paramedic"]]
   - type: ContainerFill
     containers:
       board: [ DoorElectronicsParamedic ]
+
+- type: entity
+  parent: WindoorSecure
+  id: WindoorSecureChiefJusticeLocked
+  suffix: ChiefJustice, Locked
+  components:
+  - type: AccessReader
```

Comment:
door electronics

Reply:
remove `AccessReader`

Reply:
See the rest as well

Reply:
This was also resolved, just didn't mention it